### ISSUE TITLE
Spec references are often incorrect because the biblio names do not match linking database names

### ISSUE THREAD

**Issue by domenic** [tabatkins/bikeshed#389]
```
18:06:44 <TabAtkins> I already explained it. ^_^  The Index generation sorta just hopes that the linking shortname is also the correct biblio reference.
18:06:52 <Domenic> Ahhh yes was just connecting the dots
18:06:54 <TabAtkins> Because, unfortunately, there's no explicit connection between the two databases.
18:07:09 <Domenic> ok sweet
18:07:23 <TabAtkins> That's why some of the CSS refs in the index don't work; the biblio name for them doesnt' match their shortname yet.
18:08:02 <TabAtkins> And it's annoying to mess with SpecRef there, because it does automated dumps of biblio refs from the TR.xml generated by the W3C, so if I change things it'll just revert itself later.
18:08:09 <TabAtkins> Need to figure out how to get around that.
18:08:47 <TabAtkins> Really, I probably just need to check Shepherd's spec data first; it's less complete (won't generate quite as high-quality of a biblio entry), but it's synced with the linking database names.
```

This is the root cause for https://github.com/whatwg/streams/issues/333 and at least part of https://github.com/whatwg/dom/issues/17

**Comment by tobie** [comment]
FYI, I'm syncing data from the CSSWG db on a "daily" basis (that is, when the task runner doesn't throw-up).

Here's the script I use: https://github.com/tobie/specref/blob/master/scripts/csswg.js
the DB file I use: https://github.com/tobie/specref/blob/master/scripts/csswg.js#L46
and where the script is called: https://github.com/tobie/specref/blob/master/scripts/auto-update.js#L36

I'm happy to try help you fix this issue as much as possible. I'm just unsure I probably understand where/what the problem on your side it.

**Comment by tabatkins** [comment]
Problem on my side is that not all the CSSWG specs have been published with their "proper" shortnames - many are still of the form "css3-foo" and such.  The spec-data assumes the names are constructed "properly", so it doesn't match up with the older biblio data.

**Comment by tobie** [comment]
Couldn't we just generate aliases in the meantime?

**Comment by tabatkins** [comment]
Yeah, should work.  I'll review what names we currently have in SpecRef and generate aliases in a PR for you.

(I didn't previously handle SpecRef aliases in Bikeshed's code, so it wouldn't have helped anyway; I just added that last week.)

**Comment by tabatkins** [comment]
Fixed up - all the references in Shepherd that exist in SpecRef under a different name are now fixed, either by changing Shepherd or by adding aliases to SpecRef.
